ocs_ci/ocs/must_gather/must_gather.py

```diff
@@ -134,6 +137,32 @@ def compare_running_pods(self):
             f"The difference between pod files and actual pods is: {diff}\n"
         )
 
+    def check_mg_output_disconnected_env(self):
+        """
+        Verify MG does not need to download JQ on disconnected cluster
+
+        """
+        logger.info("")
+        self.root = tempfile.mkdtemp()
+        latest_tag = ocsci_config.REPORTING.get(
+            "ocs_must_gather_latest_tag",
+            ocsci_config.REPORTING.get(
+                "default_ocs_must_gather_latest_tag",
+                ocsci_config.DEPLOYMENT["default_latest_tag"],
+            ),
+        )
+        ocs_must_gather_image = ocsci_config.REPORTING["ocs_must_gather_image"]
+        ocs_must_gather_image_and_tag = f"{ocs_must_gather_image}:{latest_tag}"
+        if ocsci_config.DEPLOYMENT.get("disconnected"):
+            ocs_must_gather_image_and_tag = mirror_image(ocs_must_gather_image_and_tag)
+        cmd = f"adm must-gather --image={ocs_must_gather_image_and_tag} --dest-dir={self.root}"
+        occli = OCP()
+        output = occli.exec_oc_cmd(command=cmd, out_yaml_format=False, timeout=900)
+        if "cannot stat 'jq'" in output:
```

Comment:
Can't this be checked in the existing test_must_gather test?

Reply:
@hnallurv done